### PR TITLE
Ac/fix monitoring

### PR DESCRIPTION
**Pull request by acabrele**
- Changes to allow monitoring to work in a multi cluster VPN solution
- Move monitoring to incubator folder
- Update to latest influx grafana reports
- Remove references in tests

### CODE REVIEW

File: stable/README.md [hunk 1]
```diff
@@ -151,7 +151,10 @@ In a nutshell the order of installation is:
 
 - **transparent-hugepage** ([documentation](transparent-hugepage/README.md))
 - **admin** ([documentation](admin/README.md))
-- **monitoring-influx** ([documentation](monitoring-influx/README.md))
+<<<<<<< HEAD
```
Comment (sivanov-nuodb) [inline stable/README.md]: you may want to fix the conflict.

File: test/integration/template_resources_test.go [hunk 2]
```diff
@@ -243,10 +243,11 @@ func TestPullSecretsRenderAllNuoDB(t *testing.T) {
 
 	helm.RenderTemplate(t, options, "../../stable/transparent-hugepage", []string{"templates/daemonset.yaml"})
 
-	helm.RenderTemplate(t, options, "../../stable/monitoring-influx", []string{"templates/deployment.yaml"})
-
+<<<<<<< HEAD
```
Comment (sivanov-nuodb) [inline test/integration/template_resources_test.go]: Here also

File: stable/README.md (hunk 1 above)
Comment (acabrele) [reply stable/README.md]: Thanks , sorted (I think)

File: test/integration/template_resources_test.go (hunk 2 above)
Comment (acabrele) [reply test/integration/template_resources_test.go]: Thanks , sorted (I think)

**Pull request merged**